src/main/resources/org/jenkinsci/plugins/workflow/support/actions/Messages.properties

```diff
@@ -0,0 +1 @@
+Workspaces=Workspaces
```

Comment:
Findbugs wants these to start with a lowercase letter (I always forget this myself as well):

> The method name org.jenkinsci.plugins.workflow.support.actions.Messages.Workspaces() doesn't start with a lower case letter [org.jenkinsci.plugins.workflow.support.actions.Messages] At Messages.java:[line 35] NM_METHOD_NAMING_CONVENTION

Actually though, should this file be deleted? It doesn't look like it's used anywhere.